tests/smoke/0001-custom.sh

```diff
@@ -0,0 +1,2 @@
+# just echo for this test
+echo "OK"
```

Comment:
do we need this extra file for just an echo?

Reply:
Yes, we need a `bash` script to test the `bash` assertion. Could be empty also, but I think have just an `echo` is better.